package.json

```diff
@@ -24,40 +24,40 @@
     "sanity"
   ],
   "dependencies": {
-    "@sanity/base": "^2.34.0",
+    "@sanity/base": "^2.35.2",
     "@sanity/block-content-to-html": "^2.0.0",
     "@sanity/block-content-to-hyperscript": "^3.0.0",
     "@sanity/block-content-to-react": "^3.0.0",
     "@sanity/client": "^3.4.1",
     "@sanity/components": "^2.14.0",
-    "@sanity/core": "^2.34.0",
-    "@sanity/dashboard": "^2.34.0",
-    "@sanity/default-layout": "^2.34.0",
-    "@sanity/default-login": "^2.34.0",
-    "@sanity/desk-tool": "^2.34.0",
+    "@sanity/core": "^2.35.0",
+    "@sanity/dashboard": "^2.35.2",
+    "@sanity/default-layout": "^2.35.2",
+    "@sanity/default-login": "^2.35.2",
+    "@sanity/desk-tool": "^2.35.2",
     "@sanity/production-preview": "^2.33.2",
-    "@sanity/ui": "^0.38.0",
-    "@sanity/vision": "^2.34.0",
-    "express": "^4.18.1",
-    "groq": "^2.29.3",
+    "@sanity/ui": "^0.37.22",
```

Comment:
Denne går ned fra 0.38 til 0.37?

Reply:
🤔 
Oppgraderte bare til Latest
![image](https://user-images.githubusercontent.com/17828446/203351323-941b45e1-3050-4f3d-a2f9-2696ce29e052.png)

Ser 0.38 kom før 0.37. Hvilken synes du vi skal gå for?
![image](https://user-images.githubusercontent.com/17828446/203351478-1e54be4a-ab27-48b4-a934-ccb75931ac03.png)

Reply:
Hæ så rart. Sier de noe om det i changelog/release info?

Reply:
Klarer ikke å se at det er noen changelog. Tenker det burde være ganske trygt å gå for latest

https://www.npmjs.com/package/@sanity/ui